pkgs/applications/networking/cluster/linkerd/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "kubernetes-linkerd-${version}";
+  version = "stable-2.4.0";
+
+  src = fetchurl {
+    url = "https://github.com/linkerd/linkerd2/releases/download/${version}/linkerd2-cli-${version}-linux";
```

Comment:
Why not build from source instead?

Reply:
I tried, but failed miserably, was getting module error (unable to detect module path or something like that). Tried to dig through linkerd build scripts, but they are just too convoluted, so gave up and just pulled in binary for now.

Reply:
fixed